configure.ac

```diff
@@ -1370,9 +1380,9 @@ echo "  build os      = $BUILD_OS"
 echo
 echo "  CC            = $CC"
 echo "  CFLAGS        = $CFLAGS"
-echo "  CPPFLAGS      = $CPPFLAGS"
+echo "  CPPFLAGS      = $CPPFLAGS $DEBUG_CPPFLAGS $HARDENED_CPPFLAGS"
 echo "  CXX           = $CXX"
-echo "  CXXFLAGS      = $CXXFLAGS"
-echo "  LDFLAGS       = $LDFLAGS"
+echo "  CXXFLAGS      = $CXXFLAGS $DEBUG_CXXFLAGS $HARDENED_CXXFLAGS"
```

Comment:
Updated. What about something like:

```bash
DEFAULT_AM_CXXFLAGS="$CXXFLAGS $DEBUG_CXXFLAGS etc..."
AC_SUBST(DEFAULT_AM_CXXFLAGS)
# and likewise for the other vars
```

Then the configure script could print that variable, and it could be re-used in src/Makefile.am. Seems like it would be less likely to get out of sync that wya.